oak/server/logging_node.cc

```diff
@@ -52,19 +52,19 @@ void LoggingNode::Run(Handle handle) {
       bool successful_parse =
           log_msg.ParseFromArray(result.msg->data.data(), result.msg->data.size());
       if (successful_parse) {
-        LOG(INFO) << "{" << name_ << "} "
-                  << "LOG: " << oak::log::Level_Name(log_msg.level()) << " " << log_msg.file()
-                  << ":" << log_msg.line() << ": " << log_msg.message();
+        OAK_LOG(INFO) << "{" << name_ << "} "
```

Comment:
Note that this particular logging should **not** be disabled, because information flow would ensure that only public data may be reach this pseudo-node. Perhaps leave a comment for now?

Reply:
Added a comment